vpr/src/util/vpr_utils.cpp

```diff
@@ -936,20 +1021,32 @@ t_pb_graph_pin* get_pb_graph_node_pin_from_model_port_pin(const t_model_ports* m
 
     if (model_port->dir == IN_PORT) {
         if (model_port->is_clock == false) {
+            t_pb_graph_pin** input_pins;
+            if (model_port->trigg_edge == TriggeringEdge::FALLING_EDGE)
```

Comment:
Again I'm not keen on this input_pins_sec etc. code, along with the duplicate latch.  It's a lot of extra code and data to sort out if we have a positive or negative register; seems like there should be some additional data we can add to all primitives that would be simpler and cleaner.